pkg/api/server/v1alpha2/plugin/plugin_logs.go

```diff
@@ -250,8 +271,133 @@ func (s *LogPluginServer) getLokiLogs(writer io.Writer, parent string, rec *db.R
 
 }
 
+func getBlobLogs(s *LogServer, writer io.Writer, parent string, rec *db.Record) error {
+	URL, err := url.Parse(s.config.LOGGING_PLUGIN_API_URL)
+	if err != nil {
+		s.logger.Error(err)
+		return err
+	}
+
+	legacy := false
+	queryParams := URL.Query()
+
+	for k, v := range s.queryParams {
+		if k == legacyLogType && v == "true" {
+			legacy = true
+			continue
+		}
+		queryParams.Add(k, v)
+	}
+
+	logPath := map[string]string{}
+
+	ctx := context.Background()
+	bucket, err := openBucket(ctx, URL.String()+"?"+queryParams.Encode())
+	if err != nil {
+		s.logger.Errorf("error opening bucket: %s", err)
+		return err
+	}
+	defer clean(bucket, s.logger)
+
+	switch rec.Type {
+	case typePipelineRun:
+		err := errors.New("pipelinerun not supported, please use taskrun")
+		s.logger.Error(err.Error())
+		return err
+	case typeTaskRun:
+		logRec := &db.Record{}
+		if legacy {
+			logRec, err = getLogRecord(s.db, parent, rec.ResultID, rec.Name)
+			if err != nil {
+				s.logger.Debugf("error getting legacy log record: %s", err)
+			}
+			if logRec != nil && err == nil {
+				log := &v1alpha3.Log{}
+				err := json.Unmarshal(logRec.Data, log)
+				if err != nil {
+					err = fmt.Errorf("could not decode Log record: %w", err)
+					s.logger.Error(err)
```

Comment:
below you call the .Error() (`s.logger.Error(err.Error())`) method on the error while logging. I do not mind either, just keep consistent.
Also in the other instances below.

Reply:
Changed it to use `s.logger.Error(err)` to everywhere.